classify/trackprediction.py

```diff
@@ -72,16 +72,30 @@ def __init__(self, track_id, start_frame, labels, keep_all=True):
         self.labels = labels
 
     def classified_clip(
-        self, predictions, smoothed_predictions, smoothed_novelties, last_frame
+        self,
+        predictions,
+        smoothed_predictions,
+        smoothed_novelties,
+        last_frame,
+        use_sum=True,
     ):
         self.last_frame_classified = last_frame
         self.num_frames_classified = len(predictions)
         self.smoothed_predictions = smoothed_predictions
         self.predictions = predictions
         self.smoothed_novelties = smoothed_novelties
-        self.class_best_score = np.max(self.smoothed_predictions, axis=0)
-        self.max_novelty = float(max(self.smoothed_novelties))
-        self.novelty_sum = sum(self.smoothed_novelties)
+        if use_sum:
```

Comment:
What is the logic for taking this branch or not?  Is this a config thing, or based on inputs?

Reply:
at the moment any new model uses this logic, while old ones dont, but really all models should. but since we are probably about ready to cut out the other models can trim this down too